api/src/main/java/org/devfleet/crest/model/CrestMarketOrder.java

```diff
@@ -0,0 +1,113 @@
+package org.devfleet.crest.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+/**
```

Comment:
Please remove the author comment. It's in the GIT history and there will be more than one in the future.